src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3599,7 +3616,27 @@ void SimAirLoopComponent(EnergyPlusData &state,
                                   ZoneEquipFlag,
                                   sensOut,
                                   latOut);
-
+        } else if (SELECT_CASE_var == CoilSystemWater) { // 'CoilSystemCooling:Water'
+            if (CompPointer == nullptr) {
+                UnitarySystems::UnitarySys thisSys;
+                CompPointer = thisSys.factory(state, DataHVACGlobals::UnitarySys_AnyCoilType, CompName, false, 0);
+                // temporary fix for saving pointer, eventually apply to UnitarySystem 16 lines above
+                state.dataAirSystemsData->PrimaryAirSystems(airLoopNum).Branch(branchNum).Comp(compNum).compPointer = CompPointer;
+            }
+            Real64 sensOut = 0.0;
+            Real64 latOut = 0.0;
+            CompPointer->simulate(state,
+                                  CompName,
+                                  FirstHVACIteration,
+                                  AirLoopNum,
+                                  CompIndex,
+                                  HeatingActive,
+                                  CoolingActive,
+                                  OAUnitNum,
+                                  OAUCoilOutTemp,
+                                  ZoneEquipFlag,
+                                  sensOut,
+                                  latOut);
```

Comment:
Simulation calling point for "CoilSystem:Cooling:Water" object placed in airloop.